csrankings.csv

```diff
@@ -1,4 +1,7 @@
 name,affiliation,homepage,scholarid
+Xiaolei Huang,Pennsylvania State Uninversity,https://faculty.ist.psu.edu/suh972/,iTtzc1UAAAAJ
+Zihan Zhou,Pennsylvania State Uninversity,https://faculty.ist.psu.edu/zzhou/Home.html,vEcgp3AAAAAJ

```

Comment:
All of these are misspelled. "Uninversity" instead of "University". Also, you didn't check any of the boxes in the commit. Please correct.

Reply:
Thanks for your reply! I have corrected them.